app/models/concerns/fog_extensions/openstack/server.rb

```diff
@@ -39,7 +39,7 @@ def flavor_with_object
       end
 
       def created_at
-        Time.parse attributes['created']
+        Time.parse(attributes['created']).utc
```

Comment:
`set_timezone` changes the time zone that's then used in prop, which is what shows the time in OpenStack.

https://github.com/theforeman/foreman/blob/develop/app/helpers/compute_resources_vms_helper.rb#L37